pkg/controllers/metrics/pod/controller.go

```diff
@@ -58,17 +58,17 @@ var (
 	podGaugeVec = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "karpenter",
-			Subsystem: "pods",
+			Subsystem: metrics.PodSubsystem,
 			Name:      "state",
 			Help:      "Pod state is the current state of pods. This metric can be used several ways as it is labeled by the pod name, namespace, owner, node, nodepool name, zone, architecture, capacity type, instance type and pod phase.",
 		},
 		labelNames(),
 	)
-	podStartupTimeSummary = prometheus.NewSummary(
+	podStartedDurationSummary = prometheus.NewSummary(
 		prometheus.SummaryOpts{
 			Namespace:  "karpenter",
-			Subsystem:  "pods",
-			Name:       "startup_time_seconds",
+			Subsystem:  metrics.PodSubsystem,
+			Name:       "started_duration_seconds",
```

Comment:
personally, I don't know if I agree with this name change, seems a little awkward to me. Can we punt on this one for now?  
```suggestion
			Name:       "started_duration_seconds",
```

Reply:
Sure